provider/provider_test.go

```diff
@@ -106,3 +108,38 @@ func TestDiffUpdates(t *testing.T) {
 	})
 
 }
+
+func TestHashIgnoresFile(t *testing.T) {
+	baseResult, err := hashContext("./testdata/ignores/basedir", "./Dockerfile")
```

Comment:
Can we use `Dockerfile` without the `./` in a few places? The defaults do not include the dot-slash.